BEIMA.Backend/UserFunctions/GetUserList.cs

```diff
@@ -0,0 +1,44 @@
+﻿using BEIMA.Backend.MongoService;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using MongoDB.Bson.Serialization;
+using System.Collections.Generic;
+
+namespace BEIMA.Backend.UserFunctions
+{
+    /// <summary>
+    /// Handles a get request involving multiple buildlings.
```

Comment:
```suggestion
    /// Handles a get request involving multiple users.
```

Reply:
Looks like I misspelled buildings on the other endpoint lol.

Reply:
Lol thanks. I went ahead and fixed this typo, as well as the original typo in GetBuildingList.cs.